shopify_api.gemspec

```diff
@@ -34,4 +34,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency("minitest", ">= 4.0")
   s.add_development_dependency("rake")
   s.add_development_dependency("timecop")
+  s.add_development_dependency("pry")
+  s.add_development_dependency("pry-byebug")
+  s.add_development_dependency("byebug")
```

Comment:
I used to just add and remove this all the time. I wasn't sure if I should make a separate PR or sneak it into whatever I was working on.